.github/workflows/test-prs.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             const label = 'run-tests';
             github.issues.removeLabel({ owner, repo, issue_number, name: label });
   test:
-    uses: aws-amplify/amplify-ui/.github/workflows/reusable-e2e.yml@main
+    uses: ./.github/workflows/reusable-e2e.yml
```

Comment:
unrelated to this pr, but do we need to make the same change on main too?

Reply:
I think that'd need some more scrutiny, since then any branch could run their local workflow.

Let's keep this for now, let's follow up internally. (fyi, this change is safe on this branch, because this is scoped to this branch only)